dev/packages/example/cisco-1.0.0/dataset/asa/agent/input/input.yml

```diff
@@ -0,0 +1,14 @@
+{{#if input == "syslog"}}
```

Comment:
What's the templating language used in these files? 

@ruflin I think it would be nice to use the appropriate file extension for the template type (e.g. .erb, .j2, .mustache) to make it clear to editors (both the people and the tools).

Reply:
It is handlebars (reminds me off https://github.com/elastic/package-registry/issues/179). And the above is actually not valid handlebars :-( Having support for 2 inputs is coming here: https://github.com/elastic/package-registry/pull/346

I agree we should probably adjust the file extension to make it better for the tooling. If we do, I wonder if we should the same for ingest pipeline files which reference an other pipeline. The challenge I see here is that 99% of the pipelines don't use it but a very few do. Does this still mean all should have to have a handlebars suffix or do we just check for both files?

Reply:
I think it would be easier if files that use templating had an extension that indicates it. Then when writing the code that uses the modules you know when you need to preprocess the file, and then you don't need to guess or unnecessarily preprocess every file like we do today in Beats.

Reply:
I wonder if supporting both types and not process would complicate the code? What is your concern about processing files that don't need processing?

Reply:
My concern is that you need to make sure you haven't used content that requires escaping in the templating language. So if you don't need those features you just call the file `config.yml` instead of `config.yml.handlebars`. Hopefully that isn't too much of issue in most files. Maybe we just process all files through the template engine for now, but put the template extension on them now.

I'd just be happy if my editor could easily recognize that these are template type X and recommend the appropriate plugin. 😄 

<img width="1121" alt="Screen Shot 2020-04-21 at 9 43 50 PM" src="https://user-images.githubusercontent.com/4565752/79931555-42eb7f00-8419-11ea-9456-ad6fbfaf8116.png">

Reply:
Should we go with `config.yml.hbs` or `config.hbs`? I skiped `handlebars` as it is very long and `.hbs` seems to work at least in my IDE too. I kind of like the two suffix so the end goal format is also in there.

Reply:
config.yml.hbs SGTM

Reply:
@ruflin if this is an update to the format of package, please open an issue in the repository for adjusting all packages.

Reply:
I introduced the `.yml.hbs` naming in this PR: https://github.com/elastic/package-registry/pull/380

Reply:
@mtojek thanks for taking care of it. I assume this make the issue obsolete.